src/signals/incident/components/form/CategorySelect/CategorySelect.js

```diff
@@ -1,19 +1,42 @@
-import React, { useMemo, useCallback } from 'react';
+import React, { useMemo, useCallback, useState, useEffect } from 'react';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
 
 import Select from 'components/SelectInput';
 import { useSelector } from 'react-redux';
 import { makeSelectSubCategories } from 'models/categories/selectors';
+import InfoText from 'components/InfoText';
+
+const Wrapper = styled.div`
+  width: 100%;
```

Comment:
A div, by default, has a width of `100%`. Why is it necessary to set it here explicitly?

Reply:
It isn't :-), I removed it.